src/utils/fetchPolicy.ts

```diff
@@ -0,0 +1,75 @@
+import { DEFAULT_BASE_URL } from '../config'
+
+export interface GetPolicyProps {
+    serviceId: string
+    policyKey: string
+    baseUrl?: string
+    version?: 'latest' | 'next'
+}
+
+/**
+ * Fetches a policy document
+ * @param serviceId
+ * @param policyKey
+ * @param version - 'latest' or 'next'
+ * @param baseUrl
+ */
+export default async function fetchPolicy({
+    serviceId,
+    policyKey,
+    version,
+    baseUrl = DEFAULT_BASE_URL,
```

Comment:
Optional & default values so that the client does not have to be aware of the api url when calling fetchPolicy on the client side.